Tests.Unit.Core.ApplicationServices/ApplicationServices/GDPR/DataProcessingRegistrationApplicationServiceTest.cs

```diff
@@ -1671,6 +1672,58 @@ public void Update_IsOversightCompletedRemark_Returns_Not_Found()
             Test_Command_Which_Fails_With_Dpr_NotFound(id => _sut.UpdateOversightCompletedRemark(id, A<string>()));
         }
 
+        [Fact]
+        public void Can_Update_MainContract()
```

Comment:
are there two test files for the app service??? do you need to spread over two files in your case?